server/scripts/addUploadSchema.ts

```diff
@@ -6,8 +6,8 @@ import minimal from './example_schemas/minimal_upload_schema.json'
   await connectToMongoose()
 
   const schema = new SchemaModel({
-    name: 'Minimal Schema v8',
-    reference: '/Minimal/General/v8',
+    name: 'Minimal Schema v10',
```

Comment:
The test config should be updated so they use this schema (the only one we expect new users to have with this update).